orc8r/cloud/go/services/orchestrator/obsidian/handlers/handlers.go

```diff
@@ -133,6 +134,9 @@ func GetObsidianHandlers() []obsidian.Handler {
 	ret = append(ret, GetPartialEntityHandlers(ManageTierImagesPath, "tier_id", new(models2.TierImages), serdes.Entity)...)
 	ret = append(ret, GetPartialEntityHandlers(ManageTierGatewaysPath, "tier_id", new(models2.TierGateways), serdes.Entity)...)
 
+	// Tracing commands
+	ret = append(ret, ctracedh.GetHandlers()...)
```

Comment:
The model we've moved to is that individual services should run their own obsidian handlers. In this case you would attach the handlers to the ctraced service.

To get this to work, you'll need to add to the service registry file:
```
ctraced:
  ...
  port: 9118 
  echo_port: 10118
  labels:
    orc8r.io/obsidian_handlers: "true"
   annotations:
     orc8r.io/obsidian_handlers_path_prefixes: /magma/v1/networks/:network_id/tracing
```

You'll also need to specify the flag `-run_echo_server` in the supervisord.conf for the ctraced service.

Lastly, you'll need to call `obsidian.AttachHandlers(srv.EchoServer, handlers.GetObsidianHandlers())` in the `main.go` of the ctraced service. 

After this, obsidian will correctly reverse proxy calls to the ctraced service